### ISSUE TITLE
[BUG] Need to disable tailwindcss/no-custom-classname for compatibility with the container-queries plugin

### ISSUE THREAD

**Issue by maxisix** [francoismassart/eslint-plugin-tailwindcss#313]
Hi,

In our project, we use `eslint-plugin-tailwindcss` and `@tailwindcss/container-queries`. We've encountered an issue where the linter flags the `no-custom-classname` error when using the container queries. Additionally, we have use the `tw-` prefix for our class names.

Below are some examples to illustrate this:

```
{/* eslint-disable-next-line tailwindcss/no-custom-classname */}
<div className="tw-@container">exemple</div>

{/* eslint-disable-next-line tailwindcss/no-custom-classname */}
<div className="@lg:-tw-mt-2">exemple</div>
```

**Comment by imink** [comment]
Got the same issue

**Comment by francoismassart** [comment]
In Tailwind CSS v4 with the `tw` prefix that would be

```html
<div class="tw:flex tw:@container">exemple</div>
<div class="tw:@lg:-mt-2">exemple</div>
```


https://play.tailwindcss.com/xdeZwKkvKs

**Comment by francoismassart** [comment]
should be fixed in v4
